packages/tenant-discovery-nodejs/src/types.ts

```diff
@@ -0,0 +1,19 @@
+import { TenantConfig } from "supertokens-node/lib/build/recipe/multitenancy/types";
+
+export type SuperTokensPluginTenantDiscoveryPluginConfig = {
+  // List of email domains that should be restricted
+  // along with the already blacklisted ones.
+  restrictedEmailDomains?: Array<string>;
```

Comment:
Do we need this as a separate thing? Maybe add it as an overrideable function, that way ppl have a way of skipping the original impl and replacing it with/ their own list, but have an easy way of adding onto it as well.